crates/lang/ir/src/ir/storage_item/config.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018-2022 Parity Technologies (UK) Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::{
+    ast,
+    error::ExtError as _,
+};
+use syn::spanned::Spanned;
+
+/// The ink! configuration.
+#[derive(Debug, Default, PartialEq, Eq)]
+pub struct StorageItemConfig {
+    /// `true` means automatically implement all storage related traits. It is default value.
+    /// `false` disables implementing all storage traits. In some cases,
+    /// it can be helpful to override the default implementation of the trait.
```

Comment:
```suggestion
    /// If set to `true`, all storage related traits are implemented automatically,
    /// this is the default value.
    /// If set to `false`, implementing all storage traits is disabled. In some cases
    /// this can be helpful to override the default implementation of the trait.
```